pkg/trace/agent/normalizer.go

```diff
@@ -194,6 +195,19 @@ func setChunkAttributesFromRoot(chunk *pb.TraceChunk, root *pb.Span) {
 		// Older tracers set origin in the root span.
 		chunk.Origin = root.Meta[tagOrigin]
 	}
+
+	if _, ok := chunk.Tags[tagDecisionMaker]; !ok {
+		var set bool
```

Comment:
Do we ever set `set` to true?

Reply:
Yes, good catch. We need to set that in the loop.

Reply:
@knusbaum I just realized that if the unit test did not caught this issue, we might be missing a unit test for this new logic.